runtime/compiler/runtime/JitRuntime.cpp

```diff
@@ -449,12 +449,20 @@ void J9FASTCALL _jitProfileAddress(uintptr_t value, TR_LinkedListProfilerInfo<ui
 
 extern "C" {
 
+#if defined(TR_HOST_X86)
+void _jProfile32BitValue(uint32_t value, TR_HashTableProfilerInfo<uint32_t> *table)
+#else
```

Comment:
Can you explain why the x86 specialization is needed here?

Reply:
It is actually only special for win32 as the J9FASTCALL macro only expands on win32.

Reply:
win32 compiler uses cdecl, which is different than fastcall, hence the specialization